### PR TITLE
delete modal for provider

### PR DESCRIPTION
**Pull request by ldidonato**
also bring header btns back

### CODE REVIEW

File: grasa_event_locator/templates/provider.php [hunk 1]
```diff
@@ -129,17 +130,39 @@
...
+                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                          <span aria-hidden="true">&times;</span>
+                        </button>
+                      </div>
+                      <div class="modal-body">
+                       Are you sure you want to delete this Event? This cannot be undone.
+                      </div>
+                      <div class="modal-footer">
+                        <button type="button" class="btn btn-secondary" data-dismiss="modal">Never Mind</button>
+                        <button type="button" class="btn btn-danger" data-dismiss="modal" onclick="">Confirm Delete</button>
```
Comment (justwheel) [inline grasa_event_locator/templates/provider.php:155]: This still needs to get hooked up, right?

Comment (ldidonato) [reply grasa_event_locator/templates/provider.php:155]: yes, its part of the list i sent you guys - delete event on provider page. This modal is on every other delete button, I just forgot it here

Comment (leong96) [reply grasa_event_locator/templates/provider.php:155]: We should have that in a list of things to do somewhere.

**Review by leong96 — APPROVED** [review]
All changes understood.

File: grasa_event_locator/templates/provider.php (hunk 1 above)
Comment (justwheel) [reply grasa_event_locator/templates/provider.php:155]: This is tracked in #79. :slightly_smiling_face:

**Pull request merged**